contrib/completion/fish/docker.fish

```diff
@@ -324,6 +335,7 @@ complete -c docker -f -n '__fish_docker_no_subcommand' -a inspect -d 'Return low
 complete -c docker -A -f -n '__fish_seen_subcommand_from inspect' -s f -l format -d 'Format the output using the given go template.'
 complete -c docker -A -f -n '__fish_seen_subcommand_from inspect' -l help -d 'Print usage'
 complete -c docker -A -f -n '__fish_seen_subcommand_from inspect' -s s -l size -d 'Display total file sizes if the type is container.'
+complete -c docker -A -f -n '__fish_seen_subcommand_from inspect' -l type -d 'Return JSON for specified type'
```

Comment:
Also one worth updating in the actual CLI (thinking of something like "Only return objects of the specified type")